### DISCUSSION TITLE
How can I locate the file by absolute path？

### DISCUSSION THREAD

**Discussion by civetming** [eslint/eslint#14141] [Q&A]
ESLint Version: 4.13.1
Node Version: v12.18.0
npm Version: 6.14.4
Operating System: windows 10

How can I locate the file by absolute path？
The path is relative in the error messages printed in the console , so I can‘t locate the error file by the link like  the first picture below

![image](https://user-images.githubusercontent.com/44012892/109116648-da2d0b80-777b-11eb-846a-03dbf151a737.png)

Correct example
![image](https://user-images.githubusercontent.com/44012892/109116600-c5e90e80-777b-11eb-90c9-050d07278dc4.png)

**Comment by mdjermanovic** [comment]
Hi @robin-ming!

> The path is relative in the error messages printed in the console , so I can‘t locate the error file by the link like the first picture below

The console output is generated by a formatter. You can choose a different one by using the [`--format` CLI option](https://eslint.org/docs/user-guide/command-line-interface#-f-format).

Here's the list of built-in formatters: https://eslint.org/docs/user-guide/formatters/

You can also install a third-party formatter: https://www.npmjs.com/search?q=eslint%20formatter

> ESLint Version: 4.13.1

It seems that you are using a very old ESLint version. The default `stylish` formatter in the actual ESLint v7.20.0 prints absolute paths.
